src/moore/static/sass/partials/pages/_news.scss

```
@@ -117,4 +117,13 @@
 
 .post-detail {
   padding: 60px 0;
-}
\ No newline at end of file
+  ul {
+    & {
+      padding-left: 40px;
+      list-style-type: initial;
+      & > li {
+        list-style-type: initial;
+      }
+    }
+  }
```

```suggestion
  ul {
      list-style-position: inside;
      & > li {
        list-style-type: initial;
      }
    }
  }
```

`list-style-type` only needs to be set on the li elements. You can also skip the padding and use list-style-position to get back the default indentation